src/Components/Widgets/widgetTypes.ts

```diff
@@ -1,9 +1,5 @@
-export enum WidgetTypes {
-  LargeWidget = 'LargeWidget',
-  MediumWidget = 'MediumWidget',
-  SmallWidget = 'SmallWidget',
-}
+import { WidgetMapping } from '../../api/dashboard-templates';
 
-export function isWidgetType(type: string): type is WidgetTypes {
-  return Object.values(WidgetTypes).includes(type as WidgetTypes);
+export function isWidgetType(widgetMapping: WidgetMapping, type: string): boolean {
+  return Object.keys(widgetMapping).includes(type);
```

Comment:
@Hyperkid123 I'm doing a dynamic check here to ensure whatever widget we come across in the layout appears in the list we get back from the new widget mapper endpoint (to avoid having 2 places to update the types/list). I can change this to union `'favoriteServices' | 'notificationsEvents` if you'd like.

Reply:
We generally don't want to check with TS types for widget types as that would mean keeping the list in two places. I think this is good.